README.md

```diff
@@ -1 +1,29 @@
-# transmogrifiers
\ No newline at end of file
+# Transmogrifiers
+This repo contains the filters, standard library, and schema files to properly process public art json files retrieved from open data sets provided by municipalities.
+
+## Functionality
+The filter folders contains javascript files designed to process json files that are retrieved from open public art datasets into a format that is much more usable for developeres to consume. This format is defined in `schemas/public-art.json` and defines required and optional attributes that are most likely to be desired by developers. 
+
+
+## Directory Structure
+```
+project
+│   README.md   
+│
+└───filters { Filters for available public art data sets }
```

Comment:
same thing as above, make more generic. If we add bike paths after this we don't want to have to rewrite this whole thing :)